ui/src/main/ui/src/actions/actions-controller.js

```diff
@@ -1,11 +1,21 @@
 angular.module('hwk.actionsModule')
-  .controller( 'hwk.actionsController', ['$scope', '$rootScope', '$q', '$modal', 'hwk.actionsService',
-  function ($scope, $rootScope, $q, $modal, actionsService) {
+  .controller( 'hwk.actionsController', ['$scope', '$rootScope', '$q', '$modal', 'hwk.actionsService', 'Notifications',
+  function ($scope, $rootScope, $q, $modal, actionsService, Notifications) {
     'use strict';
 
     console.log("[Actions] Start: " + new Date());
     console.log("[Actions] $rootScope.selectedTenant " + $rootScope.selectedTenant);
 
+    var toastError = function (reason) {
+      console.log('[Actions] Backend error ' + new Date());
```

Comment:
Do you still want the console.logs in the production code? Or maybe console.debug instead